### PR TITLE
Adjust horizontal bar chart styles as per design

### PR DESCRIPTION
**Pull request by krkshitij**
## Current Behavior

Before

## New Behavior

After

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5280dfeda349453f42b51ae49c9d128bc322c609'>5280dfeda349453f42b51ae49c9d128bc322c609</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx [hunk 1]
```diff
@@ -76,15 +76,15 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
...
+            color: palette.neutralLight,
           };
 
           const chartDataText = this._getChartDataText(points!);
           const bars = this._createBars(points!, palette);
           const keyVal = this._uniqLineText + '_' + index;
 
           return (
-            <div key={index} className={this._classNames.items}>
+            <div key={index}>
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx:87]: Why remove class name

File: change/@fluentui-react-charting-e13576b2-4b54-41bc-a96f-c30b8ada3470.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Adjust horizontal bar chart styles as per design",
+  "packageName": "@fluentui/react-charting",
+  "email": "kumarkshitij@microsoft.com",
+  "dependentChangeType": "patch"
+}
```
Comment (AtishayMsft) [inline change/@fluentui-react-charting-e13576b2-4b54-41bc-a96f-c30b8ada3470.json:7]: Get the end result reviewed from the designer

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { FontWeights } from '@fluentui/react/lib/Styling';
+import { FontSizes, FontWeights } from '@fluentui/react/lib/Styling';
 import { IHorizontalBarChartStyleProps, IHorizontalBarChartStyles } from './HorizontalBarChart.types';
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts:2]: Is the fix applied to stacked bar chart and multi stacked bar chart as well

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts (hunk 3 above)
Comment (krkshitij) [reply packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts:2]: Will create a separate PR for that

File: change/@fluentui-react-charting-e13576b2-4b54-41bc-a96f-c30b8ada3470.json (hunk 2 above)
Comment (krkshitij) [reply change/@fluentui-react-charting-e13576b2-4b54-41bc-a96f-c30b8ada3470.json:7]: ok

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx [hunk 4]
```diff
@@ -76,16 +76,21 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
...
           const bars = this._createBars(points!, palette);
           const keyVal = this._uniqLineText + '_' + index;
+          const classNames = getClassNames(this.props.styles!, {
+            theme: this.props.theme!,
+            width: this.props.width,
+            showTriangle: !!points!.chartData![0].data,
+          });
 
           return (
-            <div key={index} className={this._classNames.items}>
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx:87]: what does this change do

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts [hunk 5]
```diff
@@ -15,14 +15,12 @@ export const getHorizontalBarChartStyles = (props: IHorizontalBarChartStyleProps
...
-      height: '32px',
-      marginTop: '5px',
-      position: 'relative',
+      marginBottom: showTriangle ? '16px' : '10px',
     },
     chart: {
       width: '100%',
-      height: '8px',
-      marginBottom: '11px',
+      height: barHeight ? barHeight : 12,
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts:22]: isn't barHeight fixed?

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx [hunk 6]
```diff
@@ -76,16 +76,21 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
...
+            color: palette.neutralLight,
           };
 
           const chartDataText = this._getChartDataText(points!);
           const bars = this._createBars(points!, palette);
           const keyVal = this._uniqLineText + '_' + index;
+          const classNames = getClassNames(this.props.styles!, {
+            theme: this.props.theme!,
+            width: this.props.width,
+            showTriangle: !!points!.chartData![0].data,
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx:88]: How does !!points!.chartData![0].data imply showTriangle

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts (hunk 5 above)
Comment (krkshitij) [reply packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts:22]: I am not sure but there is a prop for barHeight. Should I ignore it?

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx (hunk 4 above)
Comment (krkshitij) [reply packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx:87]: The margin on class `items` changes depending on the prop `showTriangle` which is specific to each item. `this._classNames` doesn't include that prop, so `getClassNames` function is called for each item with the prop to get the appropriate margin

**Pull request merged**